src/components/Messages.vue

```diff
@@ -98,13 +85,22 @@ export default {
       const container = this.$el.querySelector('#container')
       container.scrollTop = container.scrollHeight
     },
-    loadMoreMessages () {
-      this.lastSeenMsgId = this.messages[0].entry.uuid
-      this.getMessageChunk({
+    scrollToMessage (id) {
+      container.scrollTop = 0
```

Comment:
Isn't `container` out of scope here?

Reply:
yeah that's really weird. The code was working, and removing this line broke it, but that doesn't make any sense

Reply:
Why are we setting scrollTop to 0 just to set it do a different value afterwards?

Reply:
it changes the result of `messageElement.offsetParent.getBoundingClientRect().top` below